src/libs/pipeline_configuration.py

```diff
@@ -0,0 +1,283 @@
+import json
+
+from core_data_modules.cleaners import swahili, Codes 
+from core_data_modules.data_models import Scheme, validators
+from dateutil.parser import isoparse
+
+def _open_scheme(filename):
+    with open(f"code_schemes/{filename}", "r") as f:
+        firebase_map = json.load(f)
+        return Scheme.from_firebase_map(firebase_map)
+
+class CodeSchemes(object):
+    ICRAF_S01E01 = _open_scheme("icraf_s01e01.json")
+    ICRAF_S01E02 = _open_scheme("icraf_s01e02.json")
+    ICRAF_S01E03 = _open_scheme("icraf_s01e03.json")
+    ICRAF_S01E04 = _open_scheme("icraf_s01e04.json")
+    ICRAF_S01E05 = _open_scheme("icraf_s01e05.json")
+    ICRAF_S01E06 = _open_scheme("icraf_s01e06.json")
+
+    AGE = _open_scheme("age.json")
+    LIVELIHOOD = _open_scheme("livelihood.json")
+    GENDER = _open_scheme("gender.json")
+    CONSTITUENCY = _open_scheme("constituency.json")
+    ORGANIZATIONS = _open_scheme("organizations.json")
+    CURRENT_PRACTICES = _open_scheme("current_practises.json")
+    NEW_PRACTICES = _open_scheme("new_practises.json")
+    REASONS_FOR_NEW_PRACTICES = _open_scheme("reasons_for_new_practices.json")
+    UPPER_TANA_PRACTISES = _open_scheme("upper_tana_practises.json")
+    YES_NO = _open_scheme("yes_no.json")
```

Comment:
(obv. this needs to be updated once my comments on #4 are addressed)

Reply:
Updated thanks!